Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -561,6 +561,13 @@ - (void)viewWillAppear:(BOOL)animated
     // or on another device.
     [self hideInputIfNeeded];
 
+    // We need to `beginLongLivedReadTransaction` before we update our
+    // mapping in order to jump to the most recent commit.
+    [self.uiDatabaseConnection beginLongLivedReadTransaction];
```

Comment:
This seems like a good thing to do, but I'm not sure if it explains the misplaced interactions (if it was intended to).

So long as we're always *reading* from our uiDatabaseConnection to present the UI - it might be stale, but shouldn't be inconsistent. right?